client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -200,7 +200,7 @@ class GSuiteAddUsers extends React.Component {
 					return <QueryEmailForwards key={ domain.domain } domainName={ domain.domain } />;
 				} ) }
 
-				<SectionHeader label={ translate( 'Add G Suite' ) } />
+				<SectionHeader label={ translate( 'Add New User' ) } />
```

Comment:
Super minor and not a blocker.

A thought. As we can add multiple users with this interface, would it be more accurate to say **Add New Users**?

Reply:
That's a good point. I went for singular to be consistent with the label of the button that leads to this page. I will update that section header as well as this button to use plural.